### DISCUSSION TITLE
Parallel work with jQuery Validation

### DISCUSSION THREAD

**Discussion by jiaulislam** [sa-si-dev/virtual-select#167] [Q&A] [answered]
I'm trying to use this library where already implemented jQuery validation in the form. I tried to validate select box in jquery form submit but not woking. Can i have an example how do i integrate validation in jquery validate form.

TIA

**Comment by sa-si-dev** [comment]
Please share the validation plugin name

**Reply by jiaulislam** [reply]
jQuery Validation Plugin

still trying from yesterday. No success yet. I really liked your plugin

[https://jqueryvalidation.org/](url)

**Comment by sa-si-dev** [comment, marked as answer]
As I checked earlier, we can't use this validation plugin with virtual select.

For more details - https://github.com/sa-si-dev/virtual-select/discussions/68#discussioncomment-2217928

**Reply by jiaulislam** [reply]
thanks @sa-si-dev for your reply. But somehow I managed to work with your library. I have created two utility function one for adding required ( don't understand while `initVirtualSelect` why `required:true` param isn't working) and other for validating. after that I'm listening for the click event.

Btw your creation is great, my form is performing much better now with lots of data. 🎉💖

```
$(".open1").click(function() {
            virtualSelectBoxIds.forEach(triggerRequired);
            if (virtualSelectBoxIds.forEach(validateVirtualSelect)) {
                console.log('True');
            } else {
                console.log('False');
            }
            if (v.form()) {
                accordion.accordion("option", "active", 1);
                current = 1;
                getCriteriaLists(2, '#output1');
            }
        });
```
